admin-dev/themes/new-theme/js/pages/order/view/order-product-add.js

```diff
@@ -0,0 +1,164 @@
+/**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+import Router from '../../../components/router';
+import OrderViewPageMap from '../OrderViewPageMap';
+import {EventEmitter} from '../../../components/event-emitter';
+import OrderViewEventMap from './order-view-event-map';
+import OrderPrices from "./order-prices";
+
+const $ = window.$;
+
+export default class OrderProductAdd {
+  constructor() {
+    this.router = new Router();
+    this.productAddActionBtn = $(OrderViewPageMap.productAddActionBtn);
+    this.productIdInput = $(OrderViewPageMap.productAddIdInput);
+    this.combinationsBlock = $(OrderViewPageMap.productAddCombinationsBlock);
+    this.combinationsSelect = $(OrderViewPageMap.productAddCombinationsSelect);
+    this.priceTaxIncludedInput = $(OrderViewPageMap.productAddPriceTaxInclInput);
+    this.priceTaxExcludedInput = $(OrderViewPageMap.productAddPriceTaxExclInput);
+    this.taxRateInput = $(OrderViewPageMap.productAddTaxRateInput);
+    this.quantityInput = $(OrderViewPageMap.productAddQuantityInput);
+    this.availableText = $(OrderViewPageMap.productAddAvailableText);
+    this.locationText = $(OrderViewPageMap.productAddLocationText);
+    this.totalPriceText = $(OrderViewPageMap.productAddTotalPriceText);
+    this.available = null;
+    this.setupListener();
+    this.product = {};
+  }
+
+  setupListener() {
+    this.combinationsSelect.on('change', (event) => {
+      this.priceTaxExcludedInput.val($(event.currentTarget).find(':selected').data('priceTaxExcluded'));
+      this.priceTaxIncludedInput.val($(event.currentTarget).find(':selected').data('priceTaxIncluded'));
+      this.available = $(event.currentTarget).find(':selected').data('stock');
+      this.quantityInput.trigger('change');
+    });
+    this.quantityInput.on('change keyup', (event) => {
+      if (this.available === null) {
+        return;
```

Comment:
use consistent return. if you have a return here, you must return something at the end of the function

Reply:
:heavy_check_mark: